scripts/python/lib/utilities.py

```diff
@@ -1343,32 +1373,40 @@ def firewall_add_services(services):
         services = [services]
 
     fw_err = 0
-    cmd = 'systemctl status firewalld.service'
+    if 'rhel' in linux_distribution(full_distribution_name=False):
+        firewall_service = 'firewalld.service'
+        firewall_enable_cmd = 'firewall-cmd --permanent --add-service='
+        firewall_reload_cmd = 'firewall-cmd --reload'
+    elif 'ubuntu' in linux_distribution(full_distribution_name=False):
+        firewall_service = 'ufw.service'
+        firewall_enable_cmd = 'ufw allow '
+        firewall_reload_cmd = 'true'
+        return 0  # TODO: REMOVE THIS DEBUG ONLY
+    cmd = f'systemctl status {firewall_service}'
     resp, err, rc = sub_proc_exec(cmd)
     if 'Active: active (running)' in resp.splitlines()[2]:
         LOG.debug('Firewall is running')
     else:
-        cmd = 'systemctl enable firewalld.service'
+        cmd = f'systemctl enable {firewall_service}'
```

Comment:
In the software installer, the test team opened an issue that we should not enable the firewall if it is not already running since this can 'break' other applications running on the box.  So if OSinstall can run without the firewall running we should just leave it disabled. Maybe just print a msg that the firewall is disabled.

Reply:
Good catch. I meant to merge this firewall setup with the logic you've already fixed (c3dac29).